gaffer-as-a-service/gaas-eureka-server/src/main/resources/application.properties

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2016-2020 Crown Copyright
+# Copyright 2016-2022 Crown Copyright
```

Comment:
I don't think this file was made in 2016. We do date of creation til last edited.

Reply:
Changed to just say 2022 as that's when it was created